sale_order_import/wizard/sale_order_import.py

```diff
@@ -197,10 +197,13 @@ def _prepare_order(self, parsed_order, price_source):
             so_vals['date_order'] = parsed_order['date']
         for line in parsed_order['lines']:
             # partner=False because we don't want to use product.supplierinfo
-            product = bdio._match_product(
-                line['product'], parsed_order['chatter_msg'], seller=False)
-            uom = bdio._match_uom(
-                line.get('uom'), parsed_order['chatter_msg'], product)
+            if line.get('code') or line.get('id') or line.get('desc') or line.get('name'):
```

Comment:
don't we need some concept for 'unmatched product' or 'unknown product' here instead?